src/selectors.js

```diff
@@ -72,31 +92,102 @@ const _buildRecords = (ents, key, opts) => {
   return finalRecords;
 };
 
+const makeNestedJoins = ({ children, records, entities }) => {
```

Comment:
records/entities betekent eig hetzelfde, zullen we het renamen naar `allRecords` en `records`?

Reply:
✅